src/core/buffers/representation/representation_buffer.ts

```diff
@@ -498,37 +481,64 @@ export default function RepresentationBuffer<T>({
   }
 
   /**
-   * Push a given segment to a QueuedSourceBuffer.
+   * Push the initialization segment to the QueuedSourceBuffer
    * @param {Object} segment
-   * @param {Object} segmentObject
+   * @param {Object} segmentData
    * @returns {Observable}
    */
-  function pushSegment(
+  function pushInitSegment(
```

Comment:
Maybe we could move pushInitSegment and pushSegment to another file ?

Reply:
Done